server/sonar-server/src/main/java/org/sonar/server/issue/ws/SearchAction.java

```diff
@@ -172,11 +174,13 @@ public void define(WebService.NewController controller) {
       .setSince("5.5")
       .setPossibleValues(RuleType.values())
       .setExampleValue(format("%s,%s", RuleType.CODE_SMELL, RuleType.BUG));
-    action.createParam(ACTION_PLANS)
-      .setDescription("Comma-separated list of action plan keys (not names)")
+    action.createParam(DEPRECATED_ACTION_PLANS)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "DEPRECATED_ACTION_PLANS"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)